File: codeforces/low_difficulty/difficulty_900/5.gravity_flip_[405A]/GravityFlip.java
/* Author: José Rodolfo (jric2002) */
import java.util.Scanner;
import java.io.BufferedReader;
import java.io.InputStreamReader;
import java.io.IOException;
import java.util.Arrays;
public class GravityFlip {
  public static void main(String[] args) throws IOException {
    int n;
    Integer[] c;
    int i;
    Scanner input_n = new Scanner(System.in);
    n = input_n.nextInt();
    c = new Integer[n];
    BufferedReader input_a = new BufferedReader(new InputStreamReader(System.in));
    i = 0;
    for (String ai : (input_a.readLine()).split(" ")) {
      c[i] = Integer.valueOf(ai);
      i++;
    }
    Arrays.sort(c);
    for (int ai : c) {
      System.out.print(ai + " ");
    }
  }
};

File: codeforces/low_difficulty/difficulty_900/5.gravity_flip_[405A]/gravity_flip.c
#include <stdio.h>
/* Author: José Rodolfo (jric2002) */
void bubbleSort(unsigned short int [], unsigned short int);
int main() {
  unsigned short int n;
  unsigned short int c[100];
  scanf("%hu", &n);
  for (unsigned short int i = 0; i < n; i++) {
    scanf("%hu", &c[i]);
  }
  bubbleSort(c, n);
  for (unsigned short int i = 0; i < n; i++) {
    printf("%i ", c[i]);
  }
  return 0;
}
void bubbleSort(unsigned short int a[], unsigned short int size_a) {
  int tn;
  for (int i = 1; i < size_a; i++) {
    for (int j = 0; j < (size_a - i); j++) {
      if (a[j] > a[j + 1]) {
        tn = a[j + 1];
        a[j + 1] = a[j];
        a[j] = tn;
      }
    }
  }
}